src/lib/homestore.cpp

```diff
@@ -365,7 +425,7 @@ shared< VirtualDev > HomeStore::create_vdev_cb(const vdev_info& vinfo, bool load
 }
 
 uint64_t HomeStore::pct_to_size(float pct, HSDevType dev_type) const {
-    uint64_t sz = uint64_cast((pct * static_cast< double >(m_dev_mgr->total_capacity())) / 100);
+    uint64_t sz = uint64_cast((pct * static_cast< double >(m_dev_mgr->total_capacity(dev_type))) / 100);
```

Comment:
This is the most significant behavior change in the case of hybrid storage. 

Previously we calculate the size based on PCT * (HDD+SSD) ,  now it is PCT * size_of_target_device_type